README.md

```diff
@@ -1,5 +1,167 @@
+    <br> This entry point can be also used to start a single patient run in order for the user to avoid the need to manually fill in all the configuration files. This can be achieved by specifying a single suffix in the <b>SUBJECTS_TO_BE_PROCESSED</b> list (e.g. SUBJECTS_TO_BE_PROCESSED=[1]).
+    
+### Configurations
+All the configuration files are under <i>pegasus/config</i> at the top level. There are configurations specific to the patient, to the machine where the workflow is running or to the actual run. Some details about each file, are given bellow:
+- environment_config.sh
+    <br> These are the machine specific configurations. The configurations are mostly paths of the software and other variables that are needed for the pipeline environment setup.
+- patient_flow.properties
+    <br> These are the patient specific configurations. 
+- pegasus.properties
+    <br> This is a pegasus specific configuration file. It just defined the paths to sites.xml, rc.txt, tc.txt and rc_out.txt.
+- rc.txt
+    <br> Inside this file, the inputs should be defined in a key-value format, where value is the path to the input file.
+    <br> During the pipeline run, the rc.txt will be filled in with all the generated files and their paths.
+- rc_out.txt
+    <br> Using this file, the output can be structured in a similar way as the input.
+- sites.xml
+    <br> This should not change since the configurable variables will be taken from the environment.
+- tc.xml
+    <br> This should not be change since it contains the commands mapping.
+
+
+## How to rerun
+There are cases when the user is not satisfied with the obtained results. 
+Maybe the volume overlapping is not correct. Maybe more tracts or longer tracts are needed. Maybe the user has T2 scans and wants to add them. 
+<br> These are all cases that imply the need to rerun the pipeline. But in the best case scenario, the user does not need to rerun the whole pipeline again, but rerun only the wanted steps.
+<br> With Pegasus the pipeline can be rerun partially. This means that it will rerun only the steps for which the corresponding outputs are missing from the <b>rc.txt</b>
+
+### Rerun with different parameters
+This is possible with pegasus, but it is not automatized. It needs user input and attention.
+In order to rerun with different parameters, the user has to:
+- modify the parameters inside the configuration files
+- search the steps that need to rerun inside the <b>main_bnm.dax</b> and identify their output files
+- remove the files (output files above) that need to be regenerated from <b>rc.txt</b>
+- start a pipeline run using one of the entry points
+
+<br> As stated before, the <b>rc.txt</b> contains a mapping between the generated file names and their paths. In order for Pegasus to rerun a group of steps, the user has to remove their output files from <b>rc.txt</b>. During the pipeline rerun, the <b>rc.txt</b> will be once again filled in.
```

Comment:
filled in, by re-running the steps (and all their dependencies) which are meant to produce the missing resources from rc.txt.